### PR TITLE
[zh]sync deployment-models

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
@zirain could you help to check why the lint is failed?

File: content/zh/docs/ops/deployment/deployment-models/index.md [hunk 1]
```diff
@@ -327,9 +475,17 @@ Istio 还支持使用集群作为租赁单位。在这种情况下，您可以
 - 集群管理员
 - 开发者
 
-要在 Istio 中使用集群租用，请将每个集群配置为一个独立的网格。或者，您可以使用 Istio 将一组集群实现为单一租户。
-然后，每个团队可以拥有一个或多个集群，但是您可以将所有集群配置为单一网格。
-要将各个团队的网格连接在一起，可以将网格联合成一个多网格部署。
+要在 Istio 中使用集群租用，
+您需要为每个团队的集群配置自己的{{< gloss >}}控制平面{{< /gloss >}}，
```
Comment (windsonsea) [inline content/zh/docs/ops/deployment/deployment-models/index.md:479]:
```suggestion
您需要为每个团队的集群配置自己的{{< gloss >}}控制平面{{< /gloss >}}，
```
可能是这种 `{{< gloss >}}控制平面{{< /gloss >}}` 被翻译后，系统无法识别这个 term 了

**Comment by ericvn** [comment]
Did depend a few minutes on the lint failure (it's not making it to lint and actually failing in the `build`):
```
-使用{{< gloss "remote cluster" >}}从集群{{< /gloss >}}或多个同步的{{< gloss "primary cluster" > }}主集群{{< /gloss >}}。
+使用{{< gloss "remote cluster" >}}从集群{{< /gloss >}}或多个同步的{{< gloss "primary cluster" >}}主集群{{< /gloss >}}。
```

**Review by windsonsea — APPROVED** [review]
Thanks, lgtm now.

**Pull request merged**